modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -114,7 +114,7 @@ System Info:    <9>
  System UUID:                             EC22BF97-A006-4A58-6AF8-0A38DEEA122A
  Boot ID:                                 f24ad37d-2594-46b4-8830-7f7555918325
  Kernel Version:                          3.10.0-957.5.1.el7.x86_64
- OS Image:                                Red Hat CoreOS 4.0
+ OS Image:                                RHEL CoreOS 4.0
```

Comment:
@mburke5678 would you need to change this to use the attribute? I am not sure what the official OS name is?

Reply:
@vikram-redhat That string is in a clode block. The attribute won't work.